coordinatorlib/src/main/scala/com/socrata/datacoordinator/id/ResourceName.scala

```diff
@@ -0,0 +1,22 @@
+package com.socrata.datacoordinator.id
+
+import com.rojoma.json.v3.codec.{DecodeError, JsonDecode, JsonEncode}
+import com.rojoma.json.v3.ast.{JValue, JString}
+
+class ResourceName(val underlying: String) {
+  override def toString = s"ResourceName($underlying)"
+}
```

Comment:
Other places where we care about user-level identifiers like this (specifically for `ColumnName`s), DC uses the types from soql-reference instead of defining its own.